### PR TITLE
add more info in certificate details

### LINKED ISSUE #442: put more content in details certificate
# Problems
In the certificate details page, the informations is lacking.

# Solutions
Display more informations.

### PR DESCRIPTION
**Pull request by ann26**
fix #442 

screenshot from 2017-06-14 18-02-23

### CODE REVIEW

**Comment by timlinux** [comment]
Can you resolve the merge conflicts @ann26 ?

**Comment by ann26** [comment]
sure @timlinux

File: django_project/certification/templates/certificate/detail.html [hunk 1]
```diff
@@ -12,16 +12,52 @@
 
 {% block content %}
 
-    {% if certificate %}
-        <div class="row" style="text-align: center">
-            <h1>Certificate Details</h1>
-            <h3>ID: {{ certificate.certificateID }}</h3>
-        </div>
+    <style>
```
Comment (dimasciput) [inline django_project/certification/templates/certificate/detail.html]: Can you put this in css file?

Comment (ann26) [inline django_project/certification/templates/certificate/detail.html]: sure

**Pull request merged**